.github/workflows/scripts/check_ignore.py

```
@@ -0,0 +1,259 @@
+                    f"""\
+Ignore directive found at line {index}, which suppresses a custom lint rule.
+Please remove this suppression and add valid documentation for \
+the respective field(s)."""
+                )
+
+                return True
+
+    return False
+
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)